contrib/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 # Distributed under the original FontForge BSD 3-clause license
 
-add_subdirectory(cidmap)
+add_subdirectory(fontforge)
```

Comment:
Why is this reordering needed?

Reply:
Hi. I did it while I was here because I thought the eventual plan was to have directories under contrib/ that are actually… well, contrib. Not project-authored, but useful third party scripts. It felt very confusing at the time, but I submitted this so long ago I have no real opinion anymore Jeremy, so if you think it sucks I can move it all back, but that'll be more work lol.

Reply:
This is fine I guess